app/templates/_has_wants_selector.html

```diff
@@ -0,0 +1,182 @@
+                {% if ppe[name]|length > 0 %}
+                <tr>
+                    {{ active_ppe_row(ppe) }}
+                </tr>
+                {% endif %}
+                {% endfor %}
+            </tbody>
+        </table>
+    </form>
+</div>
+{% endmacro %}
+
+{% block has_wants_selector %}
+<div class="modal fade" tabindex="-1" role="dialog" id="add-ppe-modal">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span
+                        aria-hidden="true">&times;</span></button>
+                <h4 class="modal-title">Add a PPE Type</h4>
+            </div>
+            <div class="modal-body">
+                <input type="search" style="width: 100%" class="form-control" placeholder="Search for a PPE Type" id="add-ppe-type-search"/>
+                <br/>
+                <table class="table table-striped" id="add-ppe-types">
+                    <thead>
+                        <tr>
+                            <th>Image</th>
+                            <th>SKU</th>
+                            <th>Description</th>
+                            <th>Actions</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for ppe in ppe_types %}
+                        {% if ppe.wants|length > 0 or ppe.has|length > 0 %}
+                            {% set hidden = "hidden" %}
+                        {% else %}
+                            {% set hidden = "" %}
+                        {% endif %}
+                        <tr data-searchable="{{ ppe.desc }} {{ ppe.sku }}" {{ hidden }} id="add-ppe-type-result-{{ ppe.id }}">
+                        {% call ppe_row(ppe) %}
+                            {% if ppe.wants|length > 0 %}
+                                <i>You already want this PPE!</i>
+                            {% elif ppe.has|length > 0 %}
+                                <i>You already have this PPE!</i>
+                            {% else %}
+                                <button class="btn btn-primary add-ppe-type" data-id="{{ ppe.id }}" data-dismiss="modal"><i class="glyphicon glyphicon-plus"></i> Add</button>
+                                <!--
+                                    This is hacky, but it works. We don't want to include templating
+                                    logic on the client, so we render it on the server and store the
+                                    HTML in a script tag with an incorrect type attribute. This
+                                    won't be rendered but will hang out safely till the client-side
+                                    JS reads it.
+                                -->
+                                <script type="text/html" id="ppe-row-template-{{ppe.id}}">
+                                    <tr>
```

Comment:
As the comment says, this script element (with a bogus type attribute so it can't be executed) contains the HTML that will be added to the main table if the user adds this PPE type. It saves the client from having to figure out the template itself.